jdatetime.class.php

```diff
@@ -68,11 +69,25 @@ class jDateTime
      */
     public function __construct($convert = null, $jalali = null, $timezone = null)
     {
-        if ( $jalali   !== null ) self::$jalali   = (bool) $jalali;
         if ( $convert  !== null ) self::$convert  = (bool) $convert;
+        if ( $jalali   !== null ) self::$jalali   = (bool) $jalali;
         if ( $timezone !== null ) self::$timezone = $timezone;
     }
 
+     /**
+     * jDateTime::setMonthNameType
+     *
+     * Pass jalali or falaki to change the month names
+     *
+     * @author Morteza Rajabi
+     * @param $type string Month name type
+     */
+    public static function setMonthNameType($type = 'jalali')
+    {
+        if(in_array($type, ['jalali', 'falaki']))
```

Comment:
Isn’t it better to move the available types to a constant and store it at top of the class?